src/main/java/tusky/Tusky.java

```diff
@@ -0,0 +1,207 @@
+                case "bye":
+                    in.close();
+                    return;
+                case "list":
+                    listTasks();
+                    break;
+                case "mark":
+                    index = Integer.parseInt(parser.getBody()) - 1;
+                    markTask(index);
+                    break;
+                case "unmark":
+                    index = Integer.parseInt(parser.getBody()) - 1;
+                    unmarkTask(index);
+                    break;
+                case "todo":
+                    try{
+                        addTask(new ToDo("false",parser.getBody()));
+                    } catch (EmptyDescriptionException e){
+                        println(Messages.LINE.toString());
+                        printf(Messages.ERR_EMPTY_TASK_DESCRIPTION.toString(), TaskType.TODO);
+                        println(Messages.LINE.toString());
+                    }
+                    break;
+                case "event":
+                    try{
+                        addTask(new Event("false",parser.getBody(), parser.get("from"), parser.get("to")));
+                    } catch (EmptyDescriptionException e){
+                        println(Messages.LINE.toString());
+                        printf(Messages.ERR_EMPTY_TASK_DESCRIPTION.toString(), TaskType.EVENT);
+                        println(Messages.LINE.toString());
+                    }
+                    break;
+                case "deadline":
+                    try{
+                        addTask(new Deadline("false",parser.getBody(), parser.get("by")));
+                    } catch (EmptyDescriptionException e){
+                        println(Messages.LINE.toString());
+                        printf(Messages.ERR_EMPTY_TASK_DESCRIPTION.toString(), TaskType.DEADLINE);
+                        println(Messages.LINE.toString());
+                    }
+                    break;
+                case "delete":
+                    index = Integer.parseInt(parser.getBody()) - 1;
+                    deleteTask(index);
+                    break;
+                default:
+                    println(Messages.LINE.toString());
+                    println(Messages.ERR_UNKNOWN_COMMAND.toString());
+                    println(Messages.LINE.toString());
+                    break;
+                }
+            } catch (KeyNotFoundException e){
+                println(Messages.LINE.toString());
+                println(e.getMessage());
+                println(Messages.LINE.toString());
+            } catch (Exception e) {
+                println(Messages.LINE.toString());
+                println(Messages.ERR_UNKNOWN_EXCEPTION.toString() + e);
+                println(Messages.LINE.toString());
+                throw e;
+            }
+
+        }
+
+    }
+
+    // println adds indentation before each println output string
+    public static void println (String x) {
+        System.out.print("    ");
```

Comment:
I can see you have utilized this spacing a number of times, do consider assigning it to a variable and referencing it from there to minimize the use of magic literals.